system-int-test/test/cmr/system_int_test/search/service/service_search_test.clj

```diff
@@ -219,7 +220,16 @@
       ;; Combination of params
       "Combination of params"
       [service3]
-      {:native-id "svc*" :provider "PROV2" "options[native-id][pattern]" true})))
+      {:native-id "svc*" :provider "PROV2" "options[native-id][pattern]" true}
+
+      "By service type"
```

Comment:
Need to add tests for `ignore_case` and `pattern` searches.

Reply:
Added tests